test/Microsoft.Extensions.SecretManager.Tests/project.json

```diff
@@ -1,11 +1,15 @@
 {
   "buildOptions": {
     "warningsAsErrors": true,
+    "nowarn": [
+      "CS0618" // Tests using obsolete APIs are skipped.
```

Comment:
Don't add a project-wide nowarn. Use `#pragma` to disable it. Example: 
![image](https://cloud.githubusercontent.com/assets/2696087/22850012/1d2d6448-efb8-11e6-97ad-688031361ba7.png)